CHANGELOG.md

```diff
@@ -1,4 +1,42 @@
 # NGI-RNAseq
 
+## [1.0.2](https://github.com/SciLifeLab/NGI-RNAseq/releases/tag/1.0.2) - 2017-04-11
+A couple of tweaks to help the pipeline in production:
+
+* Trimming FastQ files and intermediate BAM files now not saved by default
+  * This is configurable in the config or with `--saveTrimmed` / `--saveAlignedIntermediates`
+* featureCounts merge process uses `.collect()` for better consistency
+
+## [1.0.1](https://github.com/SciLifeLab/NGI-RNAseq/releases/tag/1.0.1) - 2017-04-10
+This release includes a bugfix for the last major release relating to the strandedness of `RSEQC`.
+
+* Single end reverse is now correctly `+-,-+.`
+* Single end forward is now correctly `++, --`
+* PE forward is now correctly `-1++,1--,2+-,2-+`
+
 ## [1.0](https://github.com/SciLifeLab/NGI-RNAseq/releases/tag/1.0) - 2017-04-05
-* Initial production release 
+The pipeline has been now validated for production.
+This version includes some new features:
+
+* The output from featureCounts is now merged into a single table and supplied along side the individual reports.
+* markDuplicates JVM memory is now automatically scaled based on the process memory
+* an `html` file with results documentation is now generated and supplied amongst the results
+* It's now possible to configure the pipeline for different stranded libraries with just a simple CL flag.
+* Additional support and documentation for other platforms than Uppmax. Inluding C3SE.
+* + Numerous minor tweaks and improvements.
+
+## [0.3](https://github.com/SciLifeLab/NGI-RNAseq/releases/tag/0.3) - 2016-12-13
+The pipeline has been now validated for production.
```

Comment:
> The pipeline has been now validated for production
Seems like this line is the same as in 0.3, seems redundant. Also, shouldn't it be:
`The pipeline has now been validated for production`